service/src/main/java/uk/gov/hmcts/reform/fpl/model/ChildParty.java

```diff
@@ -0,0 +1,94 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import uk.gov.hmcts.reform.fpl.model.common.EmailAddress;
+import uk.gov.hmcts.reform.fpl.model.common.Party;
+import uk.gov.hmcts.reform.fpl.model.common.TelephoneNumber;
+
+import java.util.Date;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+public final class ChildParty extends Party {
```

Comment:
could this be Child class?

Reply:
Yes but for now, since we have multiple child types, is it not important to be specific?

Reply:
@swalker125

Reply:
I guess this will be changed when we have 2nd migration right?